### PR TITLE
Dropdown update proposal

### PR DESCRIPTION
**Pull request by alejandroxlopez**
Supports Array as options, and fixes :selected property, now it works with both, arrays and options.

### CODE REVIEW

File: src/components/Inputs/Dropdown/Dropdown.vue [hunk 1]
```diff
@@ -1,29 +1,30 @@
 <template>
-  <div
-    class="input-wrap input-select select"
+  <span
```
Comment (ibdf) [inline src/components/Inputs/Dropdown/Dropdown.vue:2]: Why span?

File: src/components/Inputs/Dropdown/Dropdown.vue [hunk 2]
```diff
@@ -1,29 +1,30 @@
 <template>
-  <div
```
Comment (ibdf) [inline src/components/Inputs/Dropdown/Dropdown.vue:2]: Why remove the default classes?

File: src/components/Inputs/Dropdown/Dropdown.vue [hunk 3]
```diff
@@ -57,10 +56,27 @@ export default {
...
-  mounted() {
+  methods: {
+    isSelected(key, option) {
+      if (this.valueKey) {
+        return option[this.valueKey] === this.value;
+      }
+      return key === this.value;
+    },
+    onChange($event) {
+      this.$nextTick(() => {
```
Comment (ibdf) [inline src/components/Inputs/Dropdown/Dropdown.vue:68]: Why is the $nextTick needed?

File: src/components/Inputs/Dropdown/Dropdown.vue (hunk 2 above)
Comment (alejandroxlopez) [reply src/components/Inputs/Dropdown/Dropdown.vue:2]: asdfasdfasdf

Comment (alejandroxlopez) [reply src/components/Inputs/Dropdown/Dropdown.vue:2]: asdfasdf

Comment (alejandroxlopez) [reply src/components/Inputs/Dropdown/Dropdown.vue:2]: asdfasdfasdf

Comment (alejandroxlopez) [reply src/components/Inputs/Dropdown/Dropdown.vue:2]: asdfasdfasdf

**Review by ibdf — APPROVED** [review]

**Comment by ibdf** [comment]
This update also adds ability to disable a dropdown option.

**Review by karissademi — APPROVED** [review]
Looks good to me.

**Pull request merged**